roles/monitoring/tasks/main.yml

```diff
@@ -0,0 +1,70 @@
+---
+# tasks file for monitoring
+
+  - name: Install Logwatch
+    apt: 
+      pkg: logwatch 
+      state: installed
+
+  - name: Configure Logwatch
+    command: logwatch --detail Medium --mailto {{ Email }} --output mail --service All --range 'between -7 days and today'  
+     
+# Installation of chkrootkit
+
+  - name: Install chkrootkit
+    apt:
+      pkg: chkrootkit
+      state: present
+
+# Create a directory for chkrootkit
+  
+  - file: 
+      path: /etc/chkrootkit 
+      state: directory
+      mode: 0755 
+
+# Update chkrootkit configuration
+ 
+  - name: Update chkrootkit config
+    template:
+      src: chkrootkit.conf.j2 
+      dest: /etc/chkrootkit/chkrootkit.conf
+      owner: root
+      group: root
+      mode: 0644
+
+# Run Chkrootkit
+
+  - name: Run chkrootkit
+    shell: chkrootkit
```

Comment:
edw ti trexeis? An thes na xrisimopoiiseis to output tou, mporeis na to kaneis register...tsekare to 'register' kai pws mporeis na to xrisimopoiiseis...mporeis na kaneis px

mplamplampla
register: output_var

- debug: var=output_var gia na deis ti exei mesa afti i variable